addons/account_payment/wizards/payment_refund_wizard.py

```diff
@@ -58,8 +58,9 @@ def _compute_amount_to_refund(self):
     @api.depends('transaction_id.provider_id', 'transaction_id.payment_method_id')
     def _compute_support_refund(self):
         for wizard in self:
-            p_support_refund = wizard.transaction_id.provider_id.support_refund
-            pm = wizard.transaction_id.payment_method_id
+            tx_sudo = wizard.transaction_id.sudo()  # needed for users without access to the provider
+            p_support_refund = tx_sudo.provider_id.support_refund
+            pm = tx_sudo.payment_method_id
```

Comment:
Pro tip: always suffix `_sudo` to variables holding a sudoed recordsets. This is auto-documenting and avoids forgetting about this sudoed state 10 lines later when someone else adds code that read a field on `pm` without realizing it's bypassing every access right check.
```suggestion
            pm_sudo = tx_sudo.payment_method_id
```

Reply:
Indeed, this is more explicit !
Thank you for the review !